docs/concepts.rst

```diff
@@ -144,6 +144,31 @@ The links are exposed as a lookup table using indices from the two organizations
 The ``result_token`` (generated when creating the mapping) is required to retrieve the results. The
 ``result_type`` should be set to ``"mapping"``.
 
+Groups Result
+~~~~~~~~~~~~~
+
+The groups result has been created for multi-party linkage, and will replace the direct mapping result
+for two parties as it contains the same information in a different format.
+
+To create a project, the project owner is also required to provide the number of data providers via
+the parameter ``"number_parties"``. The server returns one ``update_token`` per parties.
+From this point, the `i-th` party is the party who received the `i-th` update token or
+`receipt_token` received after updating the clks.
```

Comment:
I was trying to describe what defined the database indexes, which are sorely defined by the receipt tokens order. I've not seen such description in any other result types, but I thought it would be useful to state what defined the indexes.